docs/source/example_notebooks/dowhy_simple_example.ipynb

```diff
@@ -366,7 +366,7 @@
    "outputs": [],
    "source": [
     "res_subset=model.refute_estimate(identified_estimand, estimate,\n",
-    "        method_name=\"data_subset_refuter\", subset_fraction=0.9)\n",
+    "        method_name=\"data_subset_refuter\", show_progress_bar=True, subset_fraction=0.9)\n",
```

Comment:
can you add the show_progress_bar=True to random common cause and placebo treatment refuter too?